src/views/tickets/email.php

```diff
@@ -435,6 +435,23 @@
 											</tr>
 										</table>
 										<?php
+										/**
+										 * Allows inserting content after the "ticket details" section.
+										 *
+										 * @since TBD
+										 */
+										do_action( 'tribe_tickets_ticket_email_after_details', $ticket, $event );
+										?>
+										<?php if ( has_action( 'tribe_tickets_ticket_email_after_details' ) ) : ?>
```

Comment:
Not sure if I'm getting the logic here. Wouldn't `has_action` return always true given that you defined the action some lines above?

Reply:
It should only return true if something actually hooks to it.

Reply:
Going to re-test and make sure it's doing what I expect.

Reply:
Right, what I mean is that if anything else is added, even if the action hooked doesn't print things, we're adding more markup. What I meant is that maybe this markup could be part of what's inserting and not be here.

Reply:
Should the hook adding content be in charge of adding the whitespace markup? Since multiple things hooking in could add multiple renders?

Reply:
Yep - moving the addition to the templates in VE.